README.rst

```diff
@@ -53,4 +59,42 @@ Developer Install
 If you are developing ``RIMEz``, clone the repo, install the dependencies as
 above, and then do (in the repo)::
 
-  $ pip install -e .[dev]
+  $ pip install -e ".[all,dev]"
+
+
+Optional Extras
+---------------
+There are a number of optional extras that can be installed along with `RIMEz`,
```

Comment:
Because this is a `.rst` file, you need double-backticks to get inline code formatting.

Reply:
Ahhh yeah..